### DISCUSSION TITLE
Show source string description as rich text

### DISCUSSION THREAD

**Discussion by pavelhoral** [WeblateOrg/weblate#9230] [Q&A] [answered]
Is it possible to handle source string description as rich text (possibly with markdown support)? Or is this only supported for `explanation`?

Currently I am working with project that has multiple sentences and newlines in the PO's `#.` _extracted comments_ field which is being handled as a single line of text by Weblate.

**Comment by nijel** [comment, marked as answer]
There is no markdown support for this right now, only links are supported here.
